model/execution/DeliveryExecutionList.php

```diff
@@ -128,6 +131,42 @@ private function createExecution($cachedData, $extraFields)
         return $execution;
     }
 
+    private function getAdjustedTime(string $deliveryExecutionId): ?int
```

Comment:
We have very similar method in [DeliveryExecutionManagerService.php](https://github.com/oat-sa/extension-tao-proctoring/blob/master/model/execution/DeliveryExecutionManagerService.php#L117). We also have methods to check if timer adjustment is allowed and adjust time in this service. I think it would be better to move this method to the same service as well to keep related logic in one place.

Reply:
done